ods_ci/tests/Tests/1000__model_serving/1006__model_serving_ovms_on_kserve_cli.robot

```diff
@@ -0,0 +1,109 @@
+Resource          ../../Resources/OCP.resource
+Resource          ../../Resources/CLI/ModelServing/llm.resource
+Library            OpenShiftLibrary
+Suite Setup       Suite Setup
+Suite Teardown    Suite Teardown
+Test Tags         KServe-OVNS   Sanity
+
+
+*** Variables ***
+${TEST_NS}=        ovnsmodel
+${RUNTIME_NAME}=  ovns-runtime
+${USE_PVC}=    ${TRUE}
+${DOWNLOAD_IN_PVC}=    ${TRUE}
+${USE_GPU}=    ${FALSE}
+${KSERVE_MODE}=    Serverless    #RawDeployment   
+${MODEL_FORMAT}=   onnx
+${PROTOCOL}=     http
+${OVERLAY}=      ${EMPTY}
+${MODELS_BUCKET}=    ${S3.BUCKET_1}
+${INFERENCE_INPUT}=    @tests/Resources/Files/modelmesh-mnist-input.json
+${INFERENCE_INPUT_OPENVINO}=    @tests/Resources/Files/openvino-example-input.json
+
+*** Test Cases ***
+Verify User Can Serve And Query A ovns Model
+    [Documentation]    Basic tests for preparing, deploying and querying a LLM model
+    ...                using Kserve and TGIS runtime
+    [Tags]    RHOAIENG-ovns
+    ${test_namespace}=  Set Variable     ovns-test-dir
+    ${model_name}=  Set Variable    test-dir
+    Setup Test Variables    model_name=test-dir    use_pvc=${USE_PVC}    use_gpu=${USE_GPU}
+    ...    kserve_mode=${KSERVE_MODE}
+    ${EXPECTED_INFERENCE_SECURED_OUTPUT}  Set Variable    {"model_name":"${MODEL_NAME}__isvc-83d6fab7bd","model_version":"1","outputs":[{"name":"Plus214_Output_0","datatype":"FP32","shape":[1,10],"data":[-8.233053,-7.7497034,-3.4236815,12.3630295,-12.079103,17.266596,-10.570976,0.7130762,3.321715,1.3621228]}]}  #robocop: disable
+
+    Set Project And Runtime    runtime=${RUNTIME_NAME}     protocol=${PROTOCOL}     namespace=${test_namespace}
+    ...    download_in_pvc=${DOWNLOAD_IN_PVC}    model_name=${model_name}
+    ...    storage_size=10Gi
+    ${requests}=    Create Dictionary    memory=10Gi
+    Compile Inference Service YAML    isvc_name=${model_name}
+    ...    sa_name=${EMPTY}
+    ...    model_storage_uri=${storage_uri}
+    ...    model_format=${MODEL_FORMAT}    serving_runtime=${RUNTIME_NAME}
+    ...    limits_dict=${limits}    requests_dict=${requests}    kserve_mode=${KSERVE_MODE}
+
+    Deploy Model Via CLI    isvc_filepath=${INFERENCESERVICE_FILLED_FILEPATH}
+    ...    namespace=${test_namespace}
+    Wait For Pods To Be Ready    label_selector=serving.kserve.io/inferenceservice=${MODEL_NAME}
+    ...    namespace=${test_namespace}
+
+    ${pod_name}=  Get Pod Name    namespace=${test_namespace}    label_selector=serving.kserve.io/inferenceservice=${model_name}
+
+    Run Keyword If    "${KSERVE_MODE}"=="RawDeployment"
+    ...    Start Port-forwarding    namespace=${test_namespace}    pod_name=${pod_name}  local_port=8888
+    ...    remote_port=8888    process_alias=ovns-process
+
+    Run Keyword And Continue On Failure    Verify OVNS Model Inference Via CLI    ${MODEL_NAME}    ${INFERENCE_INPUT}
+    ...    ${EXPECTED_INFERENCE_SECURED_OUTPUT}    token_auth=${FALSE}  test_namespace=${test_namespace}    kserve_mode=${KSERVE_MODE}
```

Comment:
Reuse the existing `Query Model Multiple Times` or `Verify Model Inference` keyword instead of creating a new one here hardcoded to OVMS

Reply:
I gave you examples in https://github.com/red-hat-data-services/ods-ci/pull/1563/files#r1656901690